cpp/docs/BENCHMARKING.md

```diff
@@ -0,0 +1,44 @@
+# Unit Benchmarking in libcudf
```

Comment:
Do we want to mention the benchmark utilities (currently only includes the random data generator)?

Reply:
I discussed the CUDA timer utility. I'm not aware of a random data generator. I can't find it in the benchmarks directory, can you point me to it? If you wrote the utility, perhaps you can write it up? Or maybe as a followup PR?

Reply:
I even presented it at a "better code" meeting :(
I can write something up in a followup PR, since this one is almost in.

Reply:
I forgot. In any case, can you point me to the files?